internal/experimental/registry/client.go

```diff
@@ -511,3 +524,18 @@ func PushOptStrictMode(strictMode bool) PushOption {
 		operation.strictMode = strictMode
 	}
 }
+
+// Tags lists all tags for a given repository
```

Comment:
It might be useful to note that the response of tags isn't sorted.

Reply:
No longer true since https://github.com/helm/helm/pull/10527/commits/348942dfc79b193a08c700d0c69eebc6f5eef3b7. Yay. See https://github.com/helm/helm/pull/10527#discussion_r780831755